src/vm/gchelpers.cpp

```diff
@@ -256,7 +256,6 @@ inline Object* Alloc(size_t size, GC_ALLOC_FLAGS flags)
     return retVal;
 }
 
-#ifdef FEATURE_64BIT_ALIGNMENT
```

Comment:
`AllocAlign8` implementation is under `FEATURE_64BIT_ALIGNMENT` in the GC. I expect that you will see failures on x86 because of this.

Reply:
Right. The functions are available, but deeper inside the implementation is ifdef'd out.
I will revert the change that removes the alignment support in the helpers to make the diff smaller - more appropriate for a bug fix. 


---
In reply to: [297736759](https://github.com/dotnet/coreclr/pull/25403#discussion_r297736759) [](ancestors = 297736759)

Reply:
Reverted the padding code. It is for x86 only now. As it should be.

---
In reply to: [297876343](https://github.com/dotnet/coreclr/pull/25403#discussion_r297876343) [](ancestors = 297876343,297736759)